frame/bounties/src/benchmarking.rs

```diff
@@ -56,7 +56,11 @@ fn setup_bounty<T: Config<I>, I: 'static>(
 		T::DataDepositPerByte::get() * T::MaximumReasonLength::get().into();
 	let _ = T::Currency::make_free_balance_be(&caller, deposit);
 	let curator = account("curator", u, SEED);
-	let _ = T::Currency::make_free_balance_be(&curator, fee / 2u32.into());
+	let mut curator_deposit = fee;
+	if let Some(curator_min_deposit) = T::CuratorDepositMin::get() {
```

Comment:
I am not an expert when it comes to this pallet; but you are ignoring the max bound and the multiplier.

Reply:
IIUC multiplier is 100% then curator_deposit == fee. and then max bound is applied. So I am proposing to mint maximum required balance so that benchmarks does not fail in any case.